app/Filament/Resources/SystemResource/Pages/ListSystems.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filament\Resources\SystemResource\Pages;
+
+use App\Filament\Resources\SystemResource;
+use Filament\Actions;
+use Filament\Resources\Pages\ListRecords;
+
+class ListSystems extends ListRecords
+{
+    protected static string $resource = SystemResource::class;
+
+    protected function getHeaderActions(): array
+    {
+        return [
+            Actions\CreateAction::make(),
+        ];
+    }
```

Comment:
Is there a way to sort the systems list by name? Would it be reasonable to make that the default sort?

Reply:
Sortable by ID and full name (default) in latest